res/translations/strings.de.po

```diff
@@ -348,6 +296,27 @@ msgstr "Phantome"
 msgid "undead_name_2"
 msgstr "Vampire"
 
+msgid "undead_name_3"
```

Comment:
Hier habe ich erst gestutzt, bis ich verstanden habe, dass Du die Strings lediglich verschoben hast. Es wäre einfacher zu reviewen, wenn das separate commits wären. Einer, der die Zeilenlängen fixt, ein zweiter, der die Strings z.B. alphabetisch sortiert, ein dritter, der unnötige Strings löscht, etc. Code Review ist schwer, wenn in einem Commit mehrere Sachen passieren, und man tausende Zeilen auf einmal überblicken muss...

Reply:
Inzwischen hat sich an den Strings so viel getan, dass es nicht mehr einfach ist, den PR zu mergen. Kannst du deinen Branch auf den aktuellen develop Branch rebasen, und einen neuen PR machen?

Reply:
Ich will's versuchen. Ist eine große Fleißarbeit. Wahrscheinlich muss ich beim Löschen vorsichtiger sein, als ich es war, scheinen ja noch einige Altlasten in der Welt zu existieren.